ktor-client/ktor-client-cio/common/src/io/ktor/client/engine/cio/CIOEngineConfig.kt

```diff
@@ -76,7 +77,17 @@ public class EndpointConfig {
     /**
      * Maximum number of connection attempts.
      */
-    public var connectRetryAttempts: Int = 1
+    @Deprecated("Misleading name.", replaceWith = ReplaceWith("connectAttempts"))
```

Comment:
A deprecation message should state what is deprecated and/or why, and what should be done to migrate. For example,
"This is deprecated due to the misleading name. Use connectAttempts instead." or "This is deprecated in favour of connectAttempts". In spite of that there is a "replace-with" block, a deprecation message is shown to user in the build log and the IDE popup and should explain what is going on and why. So that one sees that the deprecation makes sense.

Reply:
Thanks, updated